tests/data/schema/valid-nulled-schema.json

```diff
@@ -81,7 +81,7 @@
             },
             "duration": "PT3M55S",
             "explicitLyrics": false,
-            "genre": "Deutschspr. HipHop /- Rap",
```

Comment:
Should we add another schema that includes the genre? so we have a test for when it's null and when it's not null?

Reply:
I only nulled out one `genre` here. There is another one that has a value.

Reply:
👍